kmp: update getting started (#8775)
getsentry/sentry-docs: src/platform-includes/getting-started-primer/kotlin-multiplatform.mdx

Kotlin Multiplatform lets you write code once and use it on various platforms.

> It makes building app easier and more efficient by sharing common code, while still benefitting from native capabilities.

Reviewer: ```suggestion
It makes building apps easier and more efficient by sharing common code, while still benefitting from native capabilities.
```

Sentry's Kotlin Multiplatform SDK builds on top of multiple Sentry SDKs, allowing developers to share code between platforms while being able to integrate Sentry's features into their applications.
Installing this SDK will enable native crash reporting and automatic breadcrumbs for app lifecycle and UI events.

## Prerequisites

Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.

### Cocoa SDK Version Compatibility Table

Every version of our Kotlin Multiplatform SDK is compiled with a specific version of the [Sentry Cocoa SDK](/platforms/apple/).

> Use the table below to find the recommended Sentry Cocoa SDK versions for specific versions of the Kotlin Multiplatform SDK.
> Using these specific combinations ensures the best compatibility and stability.

Reviewer: ```suggestion
Use the Kotlin Multiplatform and Cocoa SDK combinations listed in the table below to ensure the best compatibility and stability.
```
